shenyu-admin/src/main/java/org/apache/shenyu/admin/mapper/FieldMapper.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shenyu.admin.mapper;
+
+import org.apache.ibatis.annotations.Mapper;
+import org.apache.shenyu.admin.model.entity.FieldDO;
+
+/**
+ * FieldMapper.
+ */
+@Mapper
+public interface FieldMapper {
+
+    /**
+     * insert record to table.
+     *
+     * @param record the record
+     * @return insert count
+     */
+    int insert(FieldDO record);
```

Comment:
Avoid using the keyword record,The new jdk will have problems

Reply:
see :https://docs.oracle.com/en/java/javase/15/language/records.html

Reply:
> Avoid using the keyword record,The new jdk will have problems

agree

Reply:
done